front/src/common/Map/utils.ts

```diff
@@ -16,7 +16,7 @@ type OnResultSearchClickType = {
   map: MapState;
   updateExtViewport: (viewport: Partial<Viewport>) => void;
   dispatch: Dispatch<AnyAction>;
-  setSearch?: React.Dispatch<React.SetStateAction<string>>;
+  setSearch?: React.Dispatch<React.SetStateAction<{ name: string; ci: string; ch: string }>>;
```

Comment:
You could export `SearchState` type in MapSearchStation.tsx. Or maybe better to move the type either in const.ts (there are other interfaces in this file) or create a type.ts file and move it with the other interfaces.
```suggestion
  setSearch?: React.Dispatch<React.SetStateAction<SearchState>>;
```